<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-JJ6KE7B" failures="1" tests="2" name="Default test(failed)" time="0.116" errors="0" timestamp="2021-01-14T15:29:50 CST">
  <testcase classname="demo.TestNgRetryFailedDemo" name="Test2" time="0.015"/>
  <testcase classname="demo.TestNgRetryFailedDemo" name="Test3" time="0.011">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at demo.TestNgRetryFailedDemo.Test3(TestNgRetryFailedDemo.java:30)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- Test3 -->
  <testcase classname="demo.TestNgRetryFailedDemo" name="Test1" time="0.0">
    <ignored/>
  </testcase> <!-- Test1 -->
</testsuite> <!-- Default test(failed) -->
